googletest/src/gtest-port.cc

```diff
@@ -90,6 +90,7 @@
 namespace testing {
 namespace internal {
 
+#if GTEST_HAS_STREAM_REDIRECTION
```

Comment:
Instead of placing the guards here, can these variables simply be moved closer to where they are used, near CaptureStdout()/CaptureStderr(), where they would already be inside the GTEST_HAS_STREAM_REDIRECTION block?

Reply:
Absolutely!